dlrover/python/master/monitor/speed_monitor.py

```diff
@@ -43,7 +44,7 @@ def __init__(self):
 class SpeedMonitor(object):
     """Monitor the training speed by the number of batch per second"""
 
-    def __init__(self):
+    def __init__(self, error_monitor: ErrorMonitor = None):
```

Comment:
Why error_monitor in speed_monitor?  The implementation of this two monitor is parallel i think.